tests/terraform/runner/test_runner.py

```diff
@@ -55,15 +56,17 @@
 ])
 class TestRunnerValid(unittest.TestCase):
     def setUp(self) -> None:
-        self.orig_checks = resource_registry.checks
+        self.orig_checks = deepcopy(resource_registry.checks)
+        self.orig_wildcard_checks = deepcopy(resource_registry.wildcard_checks)
         self.parallelization_type = parallel_runner.type
         self.db_connector = self.db_connector
         os.environ["CHECKOV_GRAPH_FRAMEWORK"] = self.graph
         os.environ["TF_SPLIT_GRAPH"] = self.tf_split_graph
 
     def tearDown(self):
         parser_registry.context = {}
-        resource_registry.checks = self.orig_checks
+        resource_registry.checks = deepcopy(self.orig_checks)
+        resource_registry.wildcard_checks = deepcopy(self.orig_wildcard_checks)
```

Comment:
I don't think these `deepcopy` calls are needed, because once at the setup is enough
```suggestion
        resource_registry.checks = self.orig_checks
        resource_registry.wildcard_checks = self.orig_wildcard_checks
```